### PR TITLE
Add support for AVIF image in site health

### LINKED ISSUE #1152: Add AVIF support information under Site Health status tab
## Feature Description

Inform users when server lacks AVIF support, similar to the WebP status information we added in https://github.com/WordPress/performance/issues/130.

Since not every server environment supports translating JPEG uploads to AVIF format, it's worth indicating this to site administrators in an easy-to-understand way. While we already have some basic PHP extension information about image capabilities in the Site Health info tab, it may be worth adding something explicitly about AVIF in the Site Health "Status" tab.

### PR DESCRIPTION
**Pull request by itzmekhokan**
## Summary

Fixes #1152 

## Relevant technical choices

Adds a module in site health to check for AVIF image format is support and shows a warning if support is not present.
If server has AVIF support, the AVIF support warning is not shown.

## Screenshot
Screenshot 2024-04-24 at 8 57 38 PM

### CODE REVIEW

**Review by westonruter — APPROVED** [review]
LGTM

File: includes/site-health/avif-support/helper.php [hunk 1]
```diff
@@ -0,0 +1,49 @@
+<?php
+/**
+ * Helper functions used for AVIF Support.
+ *
+ * @package performance-lab
+ * @since 3.0.0
```
Comment (mukeshpanchal27) [inline includes/site-health/avif-support/helper.php]:
```suggestion
 * @since n.e.x.t
```

File: includes/site-health/avif-support/helper.php [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+ */
+
+if ( ! defined( 'ABSPATH' ) ) {
+	exit; // Exit if accessed directly.
+}
+
+/**
+ * Callback for avif_enabled test.
+ *
+ * @since 1.0.0
```
Comment (mukeshpanchal27) [inline includes/site-health/avif-support/helper.php]:
```suggestion
 * @since n.e.x.t
```

File: includes/site-health/avif-support/hooks.php [hunk 3]
```diff
@@ -0,0 +1,28 @@
+<?php
+/**
+ * Hook callbacks used for AVIF Support.
+ *
+ * @package performance-lab
+ * @since 3.0.0
```
Comment (mukeshpanchal27) [inline includes/site-health/avif-support/hooks.php]:
```suggestion
 * @since n.e.x.t
```

File: includes/site-health/avif-support/hooks.php [hunk 4]
```diff
@@ -0,0 +1,28 @@
...
+ */
+
+if ( ! defined( 'ABSPATH' ) ) {
+	exit; // Exit if accessed directly.
+}
+
+/**
+ * Adds tests to site health.
+ *
+ * @since 1.0.0
```
Comment (mukeshpanchal27) [inline includes/site-health/avif-support/hooks.php]:
```suggestion
 * @since n.e.x.t
```

**Review by mukeshpanchal27 — COMMENTED** [review]
Thanks @itzmekhokan for the PR! Left some nit-pick.

**Review by mukeshpanchal27 — APPROVED** [review]

**Comment by itzmekhokan** [comment]
> Thanks @itzmekhokan for the PR! Left some nit-pick.

@mukeshpanchal27 Updated the points.

**Pull request merged**